src/utils/components/Consoles/components/CloudInitCredentials/CloudInitCredentialsContent.tsx

```diff
@@ -28,23 +27,27 @@ const CloudInitCredentialsContent: React.FC<CloudInitCredentialsContentProps> =
         for more information.
       </Trans>
       <div>
-        <strong>{t('User name: ')}</strong>
-        {user || CLOUD_INIT_MISSING_USERNAME}
-
-        {password && (
-          <>
-            <strong>{t(' Password: ')} </strong>
-
-            <ClipboardCopy
-              variant="inline-compact"
-              isCode
-              clickTip={t('Copied')}
-              hoverTip={t('Copy to clipboard')}
-            >
-              {password}
-            </ClipboardCopy>
-          </>
-        )}
+        {users.map((username) => (
```

Comment:
I think more suitiable is a user instead of a username as the username is the actual name. and the user?.name is better understood then username?.name